### DISCUSSION TITLE
Api timeout

### DISCUSSION THREAD

**Discussion by ultimatedrivingtours** [sveltia/sveltia-cms#154] [Q&A] [answered]
I have a repo that has 1000+ images. I'm getting the same timeout error as in https://github.com/sveltia/sveltia-cms/issues/46#issue-1865988483. When I delete the images, the cms loads correctly. 

```text
Something went wrong while executing your query. This may be the result of a timeout, or it could be a GitHub bug.
Please include 0B61:4E25F:1F4370E:2202169:66602549 when reporting this issue.
```

Didn't have the same issue with Decap. Wondering if this is a sveltia specific bug.

I have only tried this locally.

**Comment by kyoshino** [comment, marked as answer]
👋🏼 Hi there!

The API calls of Sveltia CMS are very different from Decap CMS, as Sveltia enables [GraphQL API](https://decapcms.org/docs/github-backend/#graphql-api) by default. Will figure out how to work around the issue. I think Sveltia just needs to send multiple requests, e.g. 500 files each.

> I have only tried this locally.

You mean you’re using the GitHub backend from `localhost`? (That’s possible)

**Reply by ultimatedrivingtours** [reply]
> 👋🏼 Hi there!
> 
> The API calls of Sveltia CMS are very different from Decap CMS, as Sveltia enables [GraphQL API](https://decapcms.org/docs/github-backend/#graphql-api) by default. Will figure out how to work around the issue. I think Sveltia just needs to send multiple requests, e.g. 500 files each.
> 
> > I have only tried this locally.
> 
> You mean you’re using the GitHub backend from `localhost`? (That’s possible)

I eventually tried on netlify, the issue occurred consistently. Yes, I use Github backend on localhost and live urls 😄

**Reply by ultimatedrivingtours** [reply]
I think the [500 limit](https://docs.github.com/en/graphql/overview/rate-limits-and-node-limits-for-the-graphql-api) is the issue in this case. Is it correct to conclude that Sveltia, in its current form,  can't handle large websites (500+ content & media files)?

**Reply by kyoshino** [reply]
The issue should be fixed with the latest version of Sveltia CMS. Can you please reload the page and try again?

**Reply by ultimatedrivingtours** [reply]
Thank you for the update. Works well.

**Comment by kyoshino** [comment]
Fixed the issue in [v0.34.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.34.0). Can you try again? You might need to delete the local cache database first.

1. Open [Chrome DevTools](https://developer.chrome.com/docs/devtools)
2. Select the Application tab
3. Expand IndexedDB in the sidebar
4. Select your repository name.
5. Click Delete Database button.
6. Reload the page.

![Screenshot 2024-06-05 at 3 06 41 PM](https://github.com/sveltia/sveltia-cms/assets/2929505/a96468b1-b402-44ac-b84c-e5a84ab3636c)
